runelite-client/src/main/java/net/runelite/client/plugins/specialcounter/SpecialCounterConfig.java

```diff
@@ -0,0 +1,35 @@
+package net.runelite.client.plugins.specialcounter;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	keyName = "specialAttack",
+	name = "Special Attack",
+	description = "Configuration for special attack options"
+)
+public interface SpecialCounterConfig extends Config
```

Comment:
Shouldn't all of the classes now be renamed to SpecialAttack* instead of SpecialCounter* as it's not solely a special counter anymore?